index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio: Setup and Mobile First Web App</title>
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+
+    <div class="wrapper">
+        
+        <header class="header">
+            
+                <a href="./index.html" class="my-logo">My Logo</a>
+                <nav class="navbar">
+                    <ul>
+                        <li class="menu-bar">                            
+                            <button type="button" class="bar"><i class="fa fa-bars"></i></button>
+                    </li>
+                </ul>
+            </nav>
+            
+        </header>
+        
+        <div class="container-homepage">     
+            <section id="#about" class="section-about">
+                <div class="welcome">
+                    <h1 class="name-welcome">
+                        I’m Amare <br> Glad to see <br> you!
+                    </h1>                
+                </div>
+                <div class="job-description">
+                    <p class="description-of-me">
+                        I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                    </p>
+                </div>
+                
+                <div class="lets-connect">
+                    <p class="lets-connect-statement">Let's connect</p>
+    
+    
+                    <div class="social-networks">
+                        <a href="#" class="social-network">
+                            <i class="fa fa-twitter"></i>
+                        </a>
+                        
+                        <a href="#" class="social-network">
+                            <i class="fa fa-linkedin"></i>
+                        </a>
+                        <a href="#" class="social-network">
+                            <i class="fa fa-medium"></i>
+                        </a>
+                        <a href="#" class="social-network">
+                            <i class="fa fa-github"></i>
+                        </a>
+                        <a href="#" class="social-network">
+                            <i class="fa fa-angellist"></i>
```

Comment:
Great design @amare1990 🤝

- [ ] [Optional] Kindly change this `div` into a `ul` and surround its child elements within `li`s. This is is a good semantic practice.